lib/services/devices/deviceRegistryMongoDB.js

```diff
@@ -21,18 +21,45 @@
  * please contact with::daniel.moranjimenez@telefonica.com
  */
 
-var logger = require('logops'),
-    dbService = require('../../model/dbConn'),
-    config = require('../../commonConfig'),
-    fillService = require('./../common/domain').fillService,
-    alarmsInt = require('../common/alarmManagement').intercept,
-    errors = require('../../errors'),
-    constants = require('../../constants'),
-    Device = require('../../model/Device'),
-    async = require('async'),
-    context = {
-        op: 'IoTAgentNGSI.MongoDBDeviceRegister'
-    };
+const logger = require('logops');
+const dbService = require('../../model/dbConn');
+const config = require('../../commonConfig');
+const fillService = require('./../common/domain').fillService;
+const alarmsInt = require('../common/alarmManagement').intercept;
+const errors = require('../../errors');
+const constants = require('../../constants');
+const Device = require('../../model/Device');
+const async = require('async');
+const cacheManager = require('cache-manager');
+const memoryCache = cacheManager.caching({store: 'memory', max: 1000, ttl: 10/*seconds*/});
```

Comment:
Magic numbers. MUST be config.

Reply:
In addition, I think the whole possibilty of not using cache (e.g. something like `cache: true|false` in configuration) should be provided.

Reply:
Fixed 9471f42 and subsequent pushes.